app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="app_name">KotlinMPP</string>
+
+    <string-array name="stations_array">
```

Comment:
nice :) see if in a future improvement you can have this in a list of a custom class (so you can have the key (the CRS) and the value (the name) separate), and only show the readable name while your logic takes in the key :)